src/components/static-map.js

```diff
@@ -141,6 +144,9 @@ export default class StaticMap extends PureComponent {
       canvas.style.outline = 'none';
     }
 
+    // Attach optional onLoad function
+    map.once('load', () => this.props.onLoad());
```

Comment:
I could be wrong, but it doesn't seem like Mapbox is passing back any param to the callback of this event. At least not that they documented anyway. https://www.mapbox.com/mapbox-gl-js/api/#map.event:load